guides/micronaut-microservices-distributed-tracing-zipkin/metadata.json

```diff
@@ -4,6 +4,7 @@
   "authors": ["Sergio del Amo"],
   "tags": ["distributed-tracing", "zipkin", "validation"],
   "categories": ["Distributed Tracing"],
+  "skips": [["maven", "groovy"]],
```

Comment:
Added a method to skip a given buildtool and language combination

Here currently Maven and Groovy doesn't work, but instead of skipping all maven, or all groovy, we can just skip maven + groovy